### PR TITLE
A0-2398: Add automatic clipboard clean up after copying sensitive data

### PR DESCRIPTION
**Pull request by jalooc**
image

### CODE REVIEW

File: packages/extension-ui/.eslintrc.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ module.exports = {
     'react/jsx-newline': ['warn', { prevent: true, allowMultilines: false }],
     'sort-keys': 'off',
     'react/jsx-no-bind': 'off',
+    '@typescript-eslint/no-floating-promises': 'off'
```
Comment (jalooc) [inline packages/extension-ui/.eslintrc.js:28]: This rule prevents applying some _common_ control flows.

File: packages/extension-ui/src/components/AccounCreationSuccess.tsx [hunk 2]
```diff
@@ -29,6 +35,10 @@ const AccountCreationSuccess = () => {
         <Header>
           Account created successfully!
         </Header>
+        <Info>
+          For safety reasons, your clipboard is going to be cleared
+          in {Math.ceil(CLIPBOARD_CLEANUP_TIMEOUT / 1000)} seconds.
```
Comment (youPickItUp) [inline packages/extension-ui/src/components/AccounCreationSuccess.tsx:40]: translation?

File: packages/extension-ui/.eslintrc.js (hunk 1 above)
Comment (youPickItUp) [reply packages/extension-ui/.eslintrc.js:28]: on second thought, perhaps we should reconsider :sweat_smile:

File: packages/extension-base/src/background/handlers/Extension.ts [hunk 3]
```diff
@@ -493,6 +494,12 @@ export default class Extension {
     return true;
   }
 
+  private clearClipboard (rawTimeout: unknown) {
+    const timeout = typeof rawTimeout === 'number' ? rawTimeout : parseInt(String(rawTimeout || 0));
+
+    setTimeout(() => clearClipboard(), timeout);
```
Comment (youPickItUp) [inline packages/extension-base/src/background/handlers/Extension.ts:500]:
```
setTimeout(clearClipboard, timeout);

File: packages/extension-base/src/background/handlers/Extension.ts [hunk 4]
```diff
@@ -493,6 +494,12 @@ export default class Extension {
     return true;
   }
 
+  private clearClipboard (rawTimeout: unknown) {
+    const timeout = typeof rawTimeout === 'number' ? rawTimeout : parseInt(String(rawTimeout || 0));
```
Comment (youPickItUp) [inline packages/extension-base/src/background/handlers/Extension.ts:498]: perhaps:
```
const timeout = parseInt(String(rawTimeout || 0));
```
would be good enough :sweat_smile:

Comment (youPickItUp) [reply packages/extension-base/src/background/handlers/Extension.ts:498]: not sure if you plan on changing this?

**Review by youPickItUp — APPROVED** [review]

File: packages/extension-base/src/background/handlers/Extension.ts (hunk 4 above)
Comment (jalooc) [reply packages/extension-base/src/background/handlers/Extension.ts]: Definitely! I even had it implemented, not sure how it vanished 😅 Thanks for pointing out 👍

**Review by youPickItUp — APPROVED** [review]

**Pull request merged**